lms/djangoapps/instructor/enrollment.py

```diff
@@ -137,7 +137,7 @@ def enroll_email(course_id, student_email, auto_enroll=False, email_students=Fal
     """
     previous_state = EmailEnrollmentState(course_id, student_email)
     enrollment_obj = None
-    if previous_state.user:
+    if previous_state.user and User.objects.get(email=student_email).is_active:
```

Comment:
Is there a reason that you're using `User.objects.get(email=student_email).is_active` here instead of `previous_state.user.is_active` ?

Reply:
@bradenmacdonald, `previous_state.user` is a boolean and not a `User` instance ([code](https://github.com/edx/edx-platform/blob/master/lms/djangoapps/instructor/enrollment.py#L64-L80)). That's why.

Reply:
Ahah, ok, that would explain it :)